### PR TITLE
[examples] Update Astro starter

### PR DESCRIPTION
**Pull request by leerob**
Deployed https://astro.vercel.app/image.

### CODE REVIEW

File: examples/astro/src/pages/image.astro [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+import { Image } from 'astro:assets';
+import astroLogo from '../assets/logo.png';
+---
+
+<Image src={astroLogo} alt="Astro Logo" width={50} />
```
Comment (styfle) [inline examples/astro/src/pages/image.astro]: I checked the deployed image and it looks like the default quality is `100` which is different that Next.js

That's probably okay if Astro wants a different default, but maybe we should add quality here to make sure users don't end up with huge images?

```suggestion
<Image src={astroLogo} alt="Astro Logo" width={50} quality={75} />
```

**Review by styfle — DISMISSED** [review]
Looks like lint is failing.

Also, don't forget to deploy again.

**Comment by leerob** [comment]
Ready to go again 👍

**Review by styfle — APPROVED** [review]
Looks good!

In the future, we might want to add a lock file.

**Review by cb1kenobi — APPROVED** [review]

**Pull request merged**